docs/_pages/basicassertions.md

```diff
@@ -76,3 +76,15 @@ Internally, `BeBinarySerializable` uses the [Object graph comparison](objectgrap
 theObject.Should().BeBinarySerializable<MyClass>(
     options => options.Excluding(s => s.SomeNonSerializableProperty));
 ```
+
```

Comment:
I am very open to copy-editing on the below.  I listed out all the errors just for google-ability.  If this had been there when I was getting the old error (which I included below), I would not have had to make this fix/PR  🤣